### ISSUE TITLE
Faster North Up and Phone orientation on map

### ISSUE THREAD

**Issue by ooosssay** [organicmaps/organicmaps#8871]
When pressing the compass to face north or when pressing the gps button to make the map orient itself to your phone, the speed in which it orients itself should be faster than what it is currently.

**Comment by biodranik** [comment]
It should be easy to tune the animation speed constant in the code.

**Comment by Misalf-git** [comment]
I agree that waiting for animations is generally annoying, but if the animaton is too slow/fast is probably highly subjective.

If it's not already the case, maybe animation duration set by the device OS could be used, so that it feels more familiar?

On Android it can be changed in "Developer Settings" or
`cmd settings put global animator_duration_scale 0.5`.

**Comment by biodranik** [comment]
Interesting info! What is the default value of it? Can the system animation duration be easily retrieved from the code? Can it be customized by users?

**Comment by Misalf-git** [comment]
> What is the default value of it?

The default value is `1.0`, which doesn't mean a lot, I guess, unless it's meant to be "one second". Might as well be a fraction of whatever value the OS has defined.

> Can the system animation duration be easily retrieved from the code?

I think so, but I don't actually know. I'm just a user who fiddles with settings to workaround annoyances.

> Can it be customized by users?

Yes, via System Settings after enabling "Developer Mode".

On rooted Android, I usually set these via adb or terminal.
```
animator_duration_scale                      0.5
transition_animation_scale                   0.0
window_animation_scale                       0.0
```

**Comment by RedAuburn** [comment]
animator_duration_scale can be queried: https://developer.android.com/reference/android/provider/Settings.Global#ANIMATOR_DURATION_SCALE

1.0 just means no scaling. We should multiply OM's animation speeds by it:
 5 seconds * 0.5 animation speed = 2.5 seconds

**Comment by biodranik** [comment]
Tuning animation for developer-only setting looks too geeky and narrow-cased )

**Comment by brdbkr** [comment]
iOS has a ["Reduce Motion" option in Accessibility Settings](https://developer.apple.com/design/human-interface-guidelines/accessibility#Motion) which is similar. I always turn it on.

Similarly, right now if I'm looking at a faraway location on OM and then hit the show my location button, it zooms out and then pans back to where I am. I find this very annoying; just jump back!

**Comment by hemanggs** [comment]
> Tuning animation for developer-only setting looks too geeky and narrow-cased )

Is this fix still required ? The current setting does feel normal like other maps apps
